aries-site/src/examples/components/input/RangeSelectorExample.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { Box, Text } from 'grommet';
+import { RangeSelector } from 'aries-core';
+
+import { UsageExample } from '../../../layouts';
+
+export const RangeSelectorExample = () => {
+  const [values, setValues] = useState([50, 80]);
+
+  return (
+    <UsageExample>
+      <Box gap="medium">
+        <Box direction="row" gap="medium">
+          <Text>Threshold</Text>
+          <Text weight={700}>
+            {values[0]}% - {values[1]}%
+          </Text>
+        </Box>
+        <Box direction="row" gap="medium" width="large" justify="between">
```

Comment:
```suggestion
        <Box direction="row" gap="medium" width="large" justify="between" align="center">
```
Think we need an align center for the range selector to be aligned w the end labels better.

Reply:
Done.